openapi/components/responses/userGroupMembership-list.yaml

```diff
@@ -7,6 +7,6 @@ content:
         A group within Schoolbox.
         
         This contains some of the folder's fields in this context.
-      type: object
-      properties:
+      type: array
```

Comment:
This raises an error when I attempt to build it:

```
npm run build --build-version=23.0.0-alpha1
```

raises:
```
[1] openapi/components/schemas/userGroupMembership-readableFields.yaml:1:1 at #/id
...
```

I think that this is supposed to be something like:
```
type: object
items:
  $ref: ../schemas/userGroupMembership-readableFields.yaml
```

but even that still shows something awkward for the response for the "get a user" endpoint:
![image](https://user-images.githubusercontent.com/407053/232663661-95e1b36f-8b68-4de0-b600-9a3730a8bb0c.png)

Reply:
I'm confused @armatronic.  I don't get the error when I build, and in the documentation for 'get a user', I see a different description for 'groups', that does not mention 'indexed by folder id of the groups'.
![image](https://user-images.githubusercontent.com/1623454/232672722-2c1b188b-2970-4265-9f18-2f5d4d78cf73.png)
![image](https://user-images.githubusercontent.com/1623454/232672941-6c10ce1a-26b2-4364-b5db-d1eae164d16b.png)

What are you and I doing differently?

Reply:
Also I noticed that I can't run 'gitsync develop' when using schoolbox-api-docs.

It tells me that my branch is up to date with origin/develop, but also that 'upstream' does not appear to be a git repository.

I tried fixing with `git branch --set-upstream-to...`, but that has not worked.

Any ideas?
![image](https://user-images.githubusercontent.com/1623454/232674939-98f4a885-e28a-47f2-8119-858d0a46554f.png)

Reply:
I manually edited my .git/config file to the following, and then I was able to run `git pull`, which updated many (88) files.  Once I had done that, I tried to rebuild and now I get the error you listed above, plus eight other errors all concerning userGroupMembership files.
![image](https://user-images.githubusercontent.com/1623454/232685995-749ffd68-91c8-4928-9876-dfe740819400.png)

So I think I am now seeing what you are seeing.  I also think I might be better off starting this PR from scratch now that my local branch is up to date.  I will try re-branching from develop and starting again.

Reply:
One more update for my future self.

I deleted my local hotfix-develop/SA-15481 branch, and added a [remote "upstream"] block (to match my working configuration for the schoolbox repository.  Now `gitsync develop` works, and I'm confident all is as it should be.
![image](https://user-images.githubusercontent.com/1623454/232687858-6091df72-0f8f-4002-9419-beb649f77aba.png)

Reply:
Sorry I didn't notice this earlier, but it looks like you've found yourself in the right place.